labs/ipfs/how-to/add-a-pin.mdx

```diff
@@ -0,0 +1,41 @@
+---
+meta:
+  title: How to add a pin to your data and retrieve it
+  description: This page explains how to add a pin to your data and retrieve it using Scaleway IPFS Pinning
+content:
+  h1: How to add a pin to your data and retrieve it
+  paragraph: This page explains how to add a pin to your data and retrieve it using Scaleway IPFS Pinning
+tags: ipfs-pinning ipfs-storage ipfs pinning storage labs web3 volume pin
+dates:
+  validation: 2023-06-27
+  posted: 2023-06-27
+categories:
+  - labs
+---
+
+Scaleway IPFS pinning allows you to permanently store a copy of your data from the public IPFS network on a Scaleway-owned node. That way, your data remains available and accessible even when your local machine is offline.
```

Comment:
Is the product name "IPFS" or "IPFS Pinning"? If it is the latter, I think the "P" in "pinning" should be capitalized :)